.github/workflows/canary-centos.yml

```diff
@@ -18,6 +18,9 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
+      - name: Install libgomp
```

Comment:
why do we need this

Reply:
onnxruntime depends on libgomp, but not bundled with the jar. User has to manually install this dependency